app/bundles/CoreBundle/Translations/en_US/flashes.ini

```diff
@@ -8,3 +8,8 @@ mautic.core.error.notfound="Item was not found."
 mautic.core.notice.created="<a href='%url%' data-toggle='ajax' data-menu-link='%menu_link%'><strong>%name%</strong></a> has been created!"
 mautic.core.notice.deleted="<strong>%name%</strong> has been deleted!"
 mautic.core.notice.updated="<a href='%url%' data-toggle='ajax' data-menu-link='%menu_link%'><strong>%name%</strong></a> has been updated!"
+mautic.core.language.helper.error.fetching.package="An error occurred while downloading the language package."
+mautic.core.language.helper.error.follow.redirects="The configuration of your server doesn't allow Mautic to fetch the language package. Either safe_mode or open_basedir is turned on which doesn't allow CURL request to follow locations. You can download the language package <a href='%url%'>here</a>. Unzip it and upload it to the /translations directory. The language was reset to the default one for now."
```

Comment:
Can we reword this to shorten it? I couldn't finish reading it before the flash disappeared.